src/main/java/com/imyvm/essential/mixin/LightningEntityInvoker.java

```diff
@@ -0,0 +1,12 @@
+package com.imyvm.essential.mixin;
+
+import net.minecraft.entity.LightningEntity;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(LightningEntity.class)
+public interface LightningEntityInvoker {
+    @Invoker("spawnFire")
+    public void invokespawnFire(int spreadAttempts);
```

Comment:
```suggestion
    void invokeSpawnFire(int spreadAttempts);
```

`interface` 不需要 `public`，再者命名也要符合习惯